allennlp/training/cross_validation.py

```diff
@@ -0,0 +1,317 @@
+    def __init__(
+        self,
+        n_splits: int = 5,
+        n_repeats: int = 10,
+        random_state: Optional[Union[int, RandomState]] = None,
+    ) -> None:
+        super().__init__(n_splits=n_splits, n_repeats=n_repeats, random_state=random_state)
+
+
+@CrossValidator.register("repeated_stratified_k_fold")
+class RepeatedStratifiedKFold(CrossValidator, sklearn.model_selection.RepeatedStratifiedKFold):
+    def __init__(
+        self,
+        n_splits: int = 5,
+        n_repeats: int = 10,
+        random_state: Optional[Union[int, RandomState]] = None,
+    ) -> None:
+        super().__init__(n_splits=n_splits, n_repeats=n_repeats, random_state=random_state)
+
+
+@CrossValidator.register("shuffle_split")
+class ShuffleSplit(CrossValidator, sklearn.model_selection.ShuffleSplit):
+    def __init__(
+        self,
+        n_splits: int = 10,
+        test_size: Optional[Union[float, int]] = None,
+        train_size: Optional[Union[float, int]] = None,
+        random_state: Optional[Union[int, RandomState]] = None,
+    ) -> None:
+        super().__init__(
+            n_splits=n_splits, test_size=test_size, train_size=train_size, random_state=random_state
+        )
+
+
+@CrossValidator.register("stratified_k_fold")
+class StratifiedKFold(CrossValidator, sklearn.model_selection.StratifiedKFold):
+    def __init__(
+        self,
+        n_splits: int = 5,
+        shuffle: bool = False,
+        random_state: Optional[Union[int, RandomState]] = None,
+    ) -> None:
+        super().__init__(n_splits=n_splits, shuffle=shuffle, random_state=random_state)
+
+
+@CrossValidator.register("stratified_shuffle_split")
+class StratifiedShuffleSplit(CrossValidator, sklearn.model_selection.StratifiedShuffleSplit):
+    def __init__(
+        self,
+        n_splits: int = 10,
+        test_size: Optional[Union[float, int]] = None,
+        train_size: Optional[Union[float, int]] = None,
+        random_state: Optional[Union[int, RandomState]] = None,
+    ) -> None:
+        super().__init__(
+            n_splits=n_splits, test_size=test_size, train_size=train_size, random_state=random_state
+        )
+
+
+@CrossValidator.register("time_series_split")
+class TimeSeriesSplit(CrossValidator, sklearn.model_selection.TimeSeriesSplit):
+    def __init__(self, n_splits: int = 5, max_train_size: Optional[int] = None) -> None:
+        super().__init__(n_splits=n_splits, max_train_size=max_train_size)
+
+
+logger = logging.getLogger(__name__)
+
+
+@TrainModel.register("cross_validation", constructor="from_partial_objects")
+class CrossValidateModel(TrainModel):
+    def __init__(
+        self,
+        serialization_dir: str,
+        dataset: Dataset,
+        data_loader_builder: Lazy[DataLoader],
+        model: Model,
+        trainer_builder: Lazy[Trainer],
+        cross_validator: CrossValidator,
+        batch_weight_key: str = "",
+        retrain: bool = False,
```

Comment:
In scikit-learn defaults to true, but I think it makes more sense here to default to false.